cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -68,8 +68,20 @@ spec:
     # deployed) to set rulesNamespace for all the clusters. Otherwise, you will get duplicate alerts with multiple alert definitions.
     rulesNamespace: rook-ceph
   network:
-    # toggle to use hostNetwork
-    hostNetwork: false
+    # supported provider are 'host' and 'multus'
+    #provider: multus
+    #selectors:
+      # In theory, you can have as many selectors as you want but the operator will only match the corresponding Ceph's settings with the first 2
+      # However, all selectors will be applied.
+      # You can name the selector the way you want, just keep in mind that the operator will do the following:
```

Comment:
Can you clarify what happens if they don't use `public` or `cluster`? Does that mean ceph just won't get any special behavior for OSDs to split traffic between public and cluster? Seems like it would be good to enforce this so they get expected behavior.

Reply:
If public or cluster is not used as a selector key we fail. I'll try to clarify again.